src/models/callStack.js

```diff
@@ -20,21 +20,64 @@ module.exports = class CallStack {
 
   // Attempts to remove any sensitive data that may be found within
   sanitize(content) {
-    if (typeof content !== "object") {
-      return content;
-    }
 
+    const hideString = "*****";
     let outContent = {};
+    let type = typeof content;
+
+    // Since JavaScript `typeof` will assign an array as "object" as well as null
+    // we will extend this typeof check to add those as different types, to ease
+    // the complexity of the below switch statement
+    if (Array.isArray(content)) {
+      type = "array";
+    }
+    if (content === null) {
+      type = "null";
+    }
```

Comment:
microoptimisation(?): maybe add a check for `content === "object"`? since these two cases would only happen if "object"

Reply:
Great idea, I'll make sure to add that in